.github/workflows/test.yml

```diff
@@ -21,6 +21,19 @@ jobs:
         platform: [ubuntu-latest]
     runs-on: ${{ matrix.platform }}
     steps:
+      - name: Setup Chrome
```

Comment:
This is interesting! So we were relying on Chrome being installed on the CI test VMs/Containers up until we started seeing these issues?

Reply:
Yeah, it's interesting, indeed. It didn't work when I removed it.